lib/web/apiserver.go

```diff
@@ -2072,6 +2072,34 @@ type siteSessionsGetResponse struct {
 	Sessions []session.Session `json:"sessions"`
 }
 
+func trackerToLegacySession(tracker types.SessionTracker, clusterName string) session.Session {
+	participants := tracker.GetParticipants()
+	parties := make([]session.Party, 0, len(participants))
+
+	for _, participant := range participants {
+		parties = append(parties, session.Party{
+			ID:         session.ID(participant.ID),
+			User:       participant.User,
+			ServerID:   tracker.GetAddress(),
+			LastActive: participant.LastActive,
```

Comment:
Do we need also add `RemoteAddr` ?

Reply:
That field isn't actually used anywhere now so we can just leave it empty.

Reply:
Added a comment to note that.